docs_src/microservices/configuration/Ch-Configuration.md

```diff
@@ -161,6 +190,7 @@ for edgex-core-data are:
 -   /edgex/core /1.0/edgex-core-data/Logging
 -   /edgex/core/1.0/edgex-core-data/MessageQueue
 -   /edgex/core/1.0/edgex-core-data/Registry
+-   /edgex/core/1.0/edgex-core-data/SecretStore
 -   /edgex/core/1.0/edgex-core-data/Service
 -   /edgex/core/1.0/edgex-core-data/Writable
 
```

Comment:
Should we put something in this section that suggests logging service is being deprecated and warn against EnableRemote?

Reply:
Done